### PR TITLE
[RFC] Redirect install.php

### PR DESCRIPTION
**Pull request by aschempp**
The install.php entry point is no longer necessary in Contao 4.4 because `contao/install` is always bootable (because default parameters are initialized).

### CODE REVIEW

**Review by Toflar — APPROVED** [review]

**Review by ausi — APPROVED** [review]

File: src/Command/InstallWebDirCommand.php [hunk 1]
```diff
@@ -73,6 +73,7 @@ protected function executeLocked(InputInterface $input, OutputInterface $output)
         $webDir = rtrim($baseDir, '/').'/web';
 
         $this->addFiles($webDir);
+        $this->removeInstallPhp($webDir);
```
Comment (leofeyer) [inline src/Command/InstallWebDirCommand.php:76]: Why did you add this if you are not distributing the `install.php` file anymore?

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: src/Command/InstallWebDirCommand.php (hunk 1 above)
Comment (aschempp) [reply src/Command/InstallWebDirCommand.php:76]: To remove the file after an update from Contao 4.3. Otherwise the user will get an exception because the kernel does no longer work.

**Review by leofeyer — APPROVED** [review]

File: src/Command/InstallWebDirCommand.php (hunk 1 above)
Comment (leofeyer) [reply src/Command/InstallWebDirCommand.php:76]: I see. 👍
